build/musicbrainz/scripts/start.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+set -eu
 
 eval $( perl -Mlocal::lib )
```

Comment:
This just breaks due to a [bug in Perl local::lib](https://rt.cpan.org/Public/Bug/Display.html?id=112711) that has been [fixed in 2.000019](https://metacpan.org/changes/distribution/local-lib#L34).
Here is a simple workaround until base image gets updated:

```suggestion
set -e

# liblocal-lib-perl < 2.000019 generates commands using unset variable
eval $( perl -Mlocal::lib )

set -u
```

Reply:
Fixed in 1c39dac2e8f33874089d4058bb96c0a3f1670967